.codecov.yaml

```diff
@@ -11,15 +11,17 @@ coverage:
   status:
     project:
       default:
-        enabled: yes
         target: 90%
         paths:
-          - api
```

Comment:
I guess the intent when this file was made was to only enforce for api/sdk. But I think this was getting ignored or not handled properly due to the lack of the trailing `/`. Anyways exclusion seems better unless we end up needing even more fine grained control.